packages/react-vapor/src/components/editor/EditorConstants.ts

```diff
@@ -8,4 +8,5 @@ export const CodeMirrorModes = {
     JSON: 'application/json',
     XML: 'xml',
     Python: 'python',
+    Text: 'text/javascript',

```

Comment:
Wouldn't it be better to use plain text ?

```suggestion
    Text: 'text/plain',
```